### ISSUE TITLE
Obsidian can't open the manifest file

### ISSUE THREAD

**Issue by rubncampelo** [brianpetro/obsidian-smart-connections#238]
Whenever I try to install this plugin, Obsidian sends me an error that says: "Failed to load plugin manifest."
I tried to install it with Obsidian42 - BRAT, but it displays the same error.

![smart-connections error](https://github.com/brianpetro/obsidian-smart-connections/assets/135176712/f1d696e7-3b5c-4bc9-8245-ce9218921ad7)

**Comment by naive231** [comment]
Same error here:

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/650107/acbd5366-91d9-4906-929f-43122f2b3c46)

**Comment by milo2146** [comment]
I have the same problem too!

**Comment by rspct666** [comment]
Have you found a solution to the issue you're facing with Obsidian and the plugin installation error?

**Comment by naive231** [comment]
No, not at all!

**Comment by rspct666** [comment]
I'm facing the same problem as well. I'm currently trying to upload Smart Connection to create a connection with the ChatGPT API, but this issue is preventing me from proceeding with it.

**Comment by rspct666** [comment]
I believe that they will resolve the issue as soon as possible.

**Comment by brianpetro** [comment]
@rubncampelo @rspct666 @milo2146 @naive231 

This should now be fixed.

Thank you, everyone, for your patience while this got resolved!

Explanation: I have started reorganizing things for an upcoming major update. While no updates have been shipped yet, the code/files in the GitHub repo have been changed. One of those changes was moving the `manifest.json` file to a sub-folder. This was problematic. So it has been moved back to the root of the repo.
